Test/Utils/Testenv/Testbed.Impl.ps1

```diff
@@ -149,6 +154,29 @@ class Testbed {
             }
         }
     }
+
+    [Void] SetDataIpInfo() {
+        function Get-IpInfo {
+            Param(
+                [Parameter(Mandatory=$true)] [string] $adapter
+            )
+            Invoke-Command -Session $this.GetSession() -ScriptBlock {
+                $Res = Get-NetIPAddress -ErrorAction SilentlyContinue -AddressFamily "IPv4" -InterfaceAlias $Using:adapter
+                if ($null -eq $Res) {
+                    return $Res
+                }
+                return @{
+                    IPAddress = $Res.IPAddress;
+                    PrefixLength = $Res.PrefixLength;
+                }
+            }
+        }
+
+        $this.DataIpInfo = Get-IpInfo($this.DataAdapterName)
```

Comment:
Get-IpInfo is a function, so it need to be called like this:
`$this.DataIpInfo = Get-IpInfo -adapter $this.DataAdapterName`
But maybe just make it a method that can be used in other places of code? I have in mind at least one where we need to get IP from specific adapter from testbed.

Reply:
Thanks, missed that.
About extracting this part of code: I think it should be done when it's needed not in anticiptation for future problems.